lib/active_model/entity/parsers/json.rb

```diff
@@ -14,11 +14,12 @@ def set_attribute_from_json(name, value)
         # Class-level methods.
         module ClassMethods
           def from_json(json)
+            unified_json = json.stringify_keys
```

Comment:
deep stringify keys? Or nested entities will be handled by their own from_json?

Reply:
yep, I expect so

Reply:
will `with_indifferent_access` be a better/faster alternative?